src/main/kotlin/no/nav/familie/ks/sak/api/ForvaltningController.kt

```diff
@@ -384,4 +386,17 @@ class ForvaltningController(
                 .build()
         }
     }
+
+    @PostMapping("/barnehagelister/dry-run")
+    fun kjørDryRunBarnehagelister(
```

Comment:
Kunne vurdert et litt mer beskrivende navn på metode og path. Et dry-run endepunkt med navnet barnehagelister kan jo være så mangt når man ser det inne i swagger. Er jo ikke så veldig viktig, for antar du kun har tenkt til å bruke dette for debugging, men om du tenker det skal ligge inne en stund kan du jo vurdere det 😊